Library/Homebrew/keg.rb

```diff
@@ -292,6 +292,23 @@ def version
     PkgVersion.parse(path.basename.to_s)
   end
 
+  def to_formula
+    Formulary.from_keg(self)
+  end
+
+  def installed_dependents
```

Comment:
I had the fallback code in here, but it meant we had to call `missing_deps` multiple times; see earlier conversation between me and @ilovezfs.

Happy to rename though.